scripts/coco-client

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3.7
+#!/usr/bin/env python3
```

Comment:
Unfortunately this needs to be 3.7 because I used the f-string syntax in a lot of places.
People seemed to be okay with adding a python3.7 installation to where this gets deployed, also we need 3.6+ anyways for sanic.

Reply:
that's fine, what I wanted to change is that it use `/usr/bin/env`. Otherwise it doesn't allow you to use a virtual environment. Would `#!/usr/bin/env python3.7` be ok?

Reply:
yes